tiledb/sm/storage_manager/storage_manager.h

```diff
@@ -755,18 +695,36 @@ class StorageManager {
       ArraySchema** array_schema);
 
   /**
-   * Loads the schema of an array from persistent storage into memory.
+   * Loads the latest schema of an array from persistent storage into memory.
    *
    * @param array_uri The URI path of the array.
    * @param encryption_key The encryption key to use.
    * @param array_schema The array schema to be retrieved.
    * @return Status
    */
-  Status load_array_schema(
+  Status load_array_schema_latest(
```

Comment:
Also this will be changed (probably removed) in a subsequent PR. I'd like to change all `Array*` to `tdb_shared_ptr<Array>`. That will allow me to load the latest array schema in another function that loads all schemas at once. That will eliminate one redundant (serial) loading.